src/tutorials/using-a-pathogen-repo/running-an-ingest-workflow.rst

```diff
@@ -0,0 +1,249 @@
+
+.. code-block::
+
+    genbank_accession    column_A    column_B    column_C
+    AF013415    AAAAA    BBBBB    CCCCC
+    AF372422    AAAAA    BBBBB    CCCCC
+    AY326412    AAAAA    BBBBB    CCCCC
+    AY632535    AAAAA    BBBBB    CCCCC
+    EU303241    AAAAA    BBBBB    CCCCC
+    EU074027    AAAAA    BBBBB    CCCCC
+    EU545988    AAAAA    BBBBB    CCCCC
+    NC_012532    AAAAA    BBBBB    CCCCC
+    DQ859059    AAAAA    BBBBB    CCCCC
+    JN860885    AAAAA    BBBBB    CCCCC
+
+
+2. Create a new rules file ``ingest/build-configs/tutorial/merge-metadata.smk``
+
+.. code-block::
+
+    rule merge_metadata:
+      input:
+        metadata="results/metadata.tsv",
+        additional_metadata="build-configs/tutorial/additional-metadata.tsv",
+      output:
+        merged_metadata="results/merged-metadata.tsv"
+      shell:
+        """
+        tsv-join -H \
+          --filter-file {input.additional_metadata} \
+          --key-fields "genbank_accession" \
+          --append-fields "*" \
+          --write-all "?" \
+          {input.metadata} > {output.merged_metadata}
+        """
+
+This rule uses `tsv-join <https://github.com/eBay/tsv-utils/blob/master/docs/tool_reference/tsv-join.md>`_ to merge the
+additional metadata with the metadata output from the ingest workflow.
+The records will be merged using the ``genbank_accession`` column and all fields from the ``additional-metadata.tsv``
+file will be appended to the metadata.
+Any record in the ``metadata.tsv`` that does not have a matching record in the ``additional-metadata.tsv`` will have a
+default ``?`` value in the new columns.
+
+3. Add the following to the custom config file ``ingest/build-configs/tutorial/config.yaml``
+
+.. code-block::
+
+    custom_rules:
+      - build-configs/tutorial/merge-metadata.smk
+
+The ``custom_rules`` config tells the ingest workflow to include your custom rules so that you can run them as part of the workflow.
+
+4. Run the ingest workflow again with the customized rule.
+
+.. code-block::
+
+    $ nextstrain build ingest merge_metadata --configfile build-configs/tutorial/config.yaml
+    Using profile profiles/default and workflow specific profile profiles/default for setting default command line arguments.
+    Config file config/defaults.yaml is extended by additional config specified via the command line.
+    Building DAG of jobs...
+    [...a lot of output...]
+
+5. Inspect the ``ingest/results/merged-metadata.tsv`` file to see that it includes the additional columns ``column_A``, ``column_B``, and ``column_C``.
+The records with the ``genbank_accession`` listed in the ``additional-metadata.tsv`` file should have the placeholder data in the new columns,
+while other records should have the default ``?`` value.
+
+Next steps
+==========
+
+* Run the `zika phylogenetic workflow <https://github.com/nextstrain/zika/tree/main/phylogenetic>`_ with new ingested data as input
+  by running
```

Comment:
Note that this won't include any of the data we (may) have just added in "Advanced usage: Customizing the ingest workflow" because the phylo workflow doesn't know about `ingest/results/merged-metadata.tsv`

Reply:
Ah, right! I'll update to use `ingest/results/merged-metadata.tsv`.

Reply:
Ah, I remembered that I didn't want to point to the new `merged-metadata.tsv` because I didn't want to go into detail of how to add the new columns to the Auspice config of the phylogenetic workflow. That seems like it needs to be a whole other tutorial...

Reply:
Maybe just a line saying "If you've customized the ingest workflow then you may need to modify the phylo workflow to use your ingested data file if it's not `results/metadata.tsv`, and other modifications to your phylo workflow may be needed, as appropriate." Or something

Reply:
Updated in https://github.com/nextstrain/docs.nextstrain.org/pull/195/commits/f021e762f2bdab67f081658c7bb044bc77c5887b